checkr/src/models/user.ts

```diff
@@ -0,0 +1,21 @@
+import { Table, Column, Model, HasMany, IsEmail } from 'sequelize-typescript';
+import { Candidate } from './candidate';
+
+@Table
+export class User extends Model<User> {
+    @Column({ primaryKey: true, autoIncrement: true })
+    userId!: number;
+
+    @Column
+    name!: string;
+
+    @IsEmail
+    @Column
+    email!: string;
+
+    @Column
+    password!: string;
+
+    @HasMany(() => Candidate)
+    candidates!: Candidate[];
```

Comment:
hopefully, this is only for ORM purpose and this does not reflect in table, please let me know about this code.
what this will do?

Reply:
This will give us access to magic functions and also this will create foreign key in candidate table referring user table if we not defined already.

Reply:
I see sequelize-typescript handles in a different way. It does not create foreignkey. It just adds association methods.